.gitignore

```diff
@@ -9,7 +9,7 @@
 *.dylib
 
 # Executables
```

Comment:
- Thanks for the catch on the DLL. Hadn't realized that it was still in the .gitignore. 
- The empty text file is just included to track the tool version. This is how the other software was doing it. 
- I had left the RandomString.vi as is, but I agree that it is convoluted. I changed it to a slightly more straightforward way to generate a temporary name. 
- In a future update I was planning of adding the .exe and .dll back to the .gitignore, hence why I did not remove the header.

As per our discussion, it is in my plan to change it from needing to use that temporary name when I do the modification of having the code store the diff in text. I will be needing to modify the caller VIs at that time regardless.

Reply:
If your intent is to add the .exe and .dll back to the .gitignore in a future pull request, let's remove this header for now (in case such a future pull request is delayed for any reason).